databases/week3/querries.sql

```diff
@@ -0,0 +1,95 @@
+--Meals Querries
+--Get all meals
+Select*
+from meal;
+--Add a new meal 
+insert into meal
+values(6, 'A Vegetarian Meal', '2 Salad, 2 Soup, 2 Starter, 2 Main Dish, 1 Desert', 'south-west', '2020-10-16', 10, 500, '2020-10-20');
+--Get a meal with any id, fx 1
+Select*
+from meal
+where id = 3;
+--Update a meal with any id, fx 1. Update any attribute fx the title or multiple attributes
+update meal
+set `price
+` = '750', `created_date` = '2020-10-05' where id= 4 ;
+--Delete a meal with any id, fx 1
+delete from meal where id = 6;
+
+
+--Reservation Querries
+
+--Get all reservations
+Select*
+from reservation;
+--Add a new reservations 
+insert into reservation
+values(6, 3, 4, '2020-10-14', '54986327', 'damie', 'damie@hotmail.com');
```

Comment:
here also like I said for meal table you have to provide the columns name.

Reply:
but I can do it without as well because I already have the table